src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,45 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService = (CinemaHallService) injector
+                .getInstance(CinemaHallService.class);
+        CinemaHall vipCinemaHall = new CinemaHall();
+        vipCinemaHall.setCapacity(100);
+        vipCinemaHall.setDescription("VIP cinema hall");
+        cinemaHallService.add(vipCinemaHall);
+        System.out.println(cinemaHallService.get(vipCinemaHall.getId()));
+        cinemaHallService.getAll().forEach(System.out::println);
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(fastAndFurious);
+        movieSession.setCinemaHall(vipCinemaHall);
+        movieSession.setShowTime(LocalDateTime.of(2022, 12, 20, 10, 30));
```

Comment:
Let's create variable for LocalDate now and then change it LocalDateTime by specifying hours and minutes.
`<LocalDate>.atTime(...)/.atStartOfDay()`
`LocalDateTime.of(<LocalDate>, LocalTime.MIN/MDNIGHT/etc.)`

Reply:
But if I want to find movie session on a specific day and time ?

Reply:
you are finding the all sessions within specified **day** only. that's what described in the definition and that's what [review comment](https://github.com/mate-academy/hibernate-create-movie-session-hw/pull/393#discussion_r1041366581) about